### PR TITLE
Sean dev

### PR DESCRIPTION
**Pull request by sean-darsie**
Created test file for librarian controller. Added basic tests to ensure the response from the controller in cases of success and failure from the service layer.

### CODE REVIEW

File: src/main/java/com/ss/training/lms/controller/LibrarianController.java [hunk 1]
```diff
@@ -117,47 +121,67 @@
 
     }
 
-    @PutMapping(value = "/lms/librarian/branches/{branchId}/books/{bookId}")
-    public ResponseEntity<String> updateCopies(@PathVariable int branchId,
+    @PutMapping(value = "/lms/librarian/branches/{branchId}/books/{bookId}",
+            produces = {
+                    MediaType.APPLICATION_JSON_VALUE,
+                    MediaType.APPLICATION_XML_VALUE
```
Comment (trevorhuis-smoothstack) [inline src/main/java/com/ss/training/lms/controller/LibrarianController.java]: The media types aren't needed with a pom dependency. I think Justin found it. The media types don't hurt, but removing them could clean up the code.

File: src/test/java/com/ss/training/lms/LibrarianControllerTest.java [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+    LibrarianService librarianService;
+	
+	@InjectMocks
+	private LibrarianController librarianController;
+	
+	@BeforeAll
+	public void setup() throws Exception
+	{
+		mockMvc = MockMvcBuilders.standaloneSetup(librarianController).build();
+	}
```
Comment (trevorhuis-smoothstack) [inline src/test/java/com/ss/training/lms/LibrarianControllerTest.java:48]: I am not sure how all this works at the moment but looks like you put in a ton of work! great stuff.

**Review by trevorhuis-smoothstack — APPROVED** [review]
Hey Sean, I don't know much about Mockito right now so it's tough to be helpful. It looks very powerful. I think we should all call and go over the changes we made in our current assignments so we can come back through the code each other wrote with more knowledge.

File: src/main/java/com/ss/training/lms/controller/LibrarianController.java (hunk 1 above)
Comment (sean-darsie) [reply src/main/java/com/ss/training/lms/controller/LibrarianController.java]: oh awesome. ok. I'll remove them then.

**Pull request merged**